game.rb

```diff
@@ -0,0 +1,50 @@
+require_relative 'environment'
+
+class Game 
+  attr_accessor :depot, :control
+  include Environment
+
+  def initialize
+    @board = { payers: {}, control: {}, status: nil}
+    @depot = Environment::Depot.new
+    @control = Environment::Control.new
+  end
+
+  def new_worker(name:)
+    worker = Environment::Worker.new(name:name, personal_data: nil,  professional_data: nil)
+    @board[:payers[name]] = worker
+
+  end
+
+  def new_mission(name:, objective: , pack: )
+
+    @control.new_mission(name:name, objective:objective, pack:pack)
+  end
+
+end
+
+game = Game.new
+
+
+#TESTS
```

Comment:
Nice! When you learn about RSpec, you will be happy because it i an amazing way to write tests.